travis/push-to-pypi.sh

```diff
@@ -3,21 +3,21 @@ set -x
 SCRIPTS_DIR=$(dirname ${BASH_SOURCE[0]})
 source "$SCRIPTS_DIR/globals.sh"
 
-TEST_PYPI_RELEASE_HINT="release to test.pypi.org"
```

Comment:
I would prefer to preserve all the earlier logic for test-pypi; only add additional case to push to test-pypi if the author is noiro-tagger.